packages/base/src/3dview/mainview.tsx

```diff
@@ -752,10 +758,13 @@ export class MainView extends React.Component<IProps, IStates> {
             threeShape,
             exported => {
               pos.postShape = exported as any;
+              resolved();
```

Comment:
it should resolve the promise at the end of this loop?

Reply:
Looks like we need some kind of `Promise.all`

Reply:
I added a `Promise.all` after the loop.